StandardCyborgNetworking/StandardCyborgNetworking/DefaultServerAPIClient.swift

```diff
@@ -220,18 +233,42 @@ public class DefaultServerAPIClient: NSObject, ServerAPIClient, URLSessionTaskDe
         }
     }
     
-    private func _updateCredentialsFromURLResponse(_ urlResponse: URLResponse) throws {
+    private func _updatePersonalCredentialsFromURLResponse(_ urlResponse: URLResponse) throws {
         guard let httpResponse = urlResponse as? HTTPURLResponse else {
             throw ServerOperationError.genericErrorString("URL response was not an HTTP response for \(urlResponse.url?.absoluteString ?? "no URL")")
         }
         
         let responseHeaders = httpResponse.allHeaderFields
-        
-        if responseHeaders[ResponseHeader.accessToken.rawValue] != nil {
-            _credentials.updateFromResponseHeaders(responseHeaders)
-//            throw ServerOperationError.genericErrorString("Sign in failed. No access-token header was returned in the response.")
+
+        if responseHeaders[ResponseHeader.client.rawValue] != nil {
+            // Create an instance here since it will be updated immediately after. An empty initialization let's
+            // us avoid needing to duplicate the logic to pull the values out of the headers.
+            // This could also be moved to an init method if desired.
+            if self._personalCredentials == nil {
+                _personalCredentials = PersonalCredentials(expiry: "", tokenType: "", accessToken: "", client: "", uid: "")
+            }
+            _personalCredentials?.updateAndPersistFromResponseHeaders(responseHeaders)
         }
     }
+
+    private func _updateTeamCredentialsFromResponseBody(_ responseBody: Any) {
+        guard
+            let dict = responseBody as? [String : Any],
+            let accessToken = dict[TeamCredentials.Key.accessToken.rawValue] as? String,
```

Comment:
Should both of these use `Key` here?

Reply:
Hmm so `Key` is the name of the enum, but the actual string value I think maps correctly? Maybe the enum should be named something else?

Reply:
Oh sorry, that's my lack of reading comprehension. I missed the `accessToken` and `apiKey` after that.